Extension/src/LanguageServer/client.ts

```diff
@@ -1980,123 +1999,128 @@ export class DefaultClient implements Client {
             return;
         }
         telemetry.logLanguageServerEvent('provideCustomConfiguration', { providerId });
+        void this.provideCustomConfigurationAsync(docUri, requestFile, replaceExisting, onFinished, provider);
+    }
 
-        return this.queueBlockingTask(async () => {
-            const tokenSource: vscode.CancellationTokenSource = new vscode.CancellationTokenSource();
-            console.log("provideCustomConfiguration");
+    private async provideCustomConfigurationAsync(docUri: vscode.Uri,requestFile: string|undefined,  replaceExisting: boolean|undefined, onFinished: () => void, provider: CustomConfigurationProvider1) {
```

Comment:
Nit: We usually have spaces around the `|`

Reply:
My preference would be to either run the formatter or keeping the formatting consistent, i.e. spaces around `|` and before `=>`, etc.